deploy/kubernetes/helm/che/templates/resource-monitor-role.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+{{- if not (contains "<" .Values.global.cheWorkspacesNamespace) }}
```

Comment:
Could you explain why do we need this ?

Reply:
This commit will be a answer. 7d56354a5efefe4ff7e9d961ee1ca5688acea478